app/backend/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import 'tsconfig-paths/register';
+import express, { Express, Request, Response } from "express";
+
+import dotenv from "dotenv";
+import morgan from 'morgan';
+import swaggerUi from 'swagger-ui-express';
+import swaggerJsdoc from "swagger-jsdoc";
+
+import registerRoutes from '@routes/index';
```

Comment:
remove /index

Reply:
temp use: "@routes": ["routes/index"], in tsconfig